kotsadm/pkg/kurl/kurl_nodes.go

```diff
@@ -2,16 +2,22 @@ package kurl
 
 import (
 	"context"
+	"crypto/tls"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"math"
+	"net/http"
 	"strconv"
 
 	"github.com/pkg/errors"
-	"github.com/replicatedhq/kots/kotsadm/pkg/k8s"
 	"github.com/replicatedhq/kots/kotsadm/pkg/kurl/types"
 	"github.com/replicatedhq/kots/kotsadm/pkg/logger"
 	v1 "k8s.io/api/core/v1"
 	kuberneteserrors "k8s.io/apimachinery/pkg/api/errors"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/client-go/kubernetes"
+	"k8s.io/kubernetes/pkg/kubelet/apis/stats/v1alpha1"
```

Comment:
@laverya will you import this with an alias?  `v1alpha1` will be ambiguous in the code.  Maybe `statsv1alpha1` or `kubeletv1alpha1`